checks.d/mongo.py

```diff
@@ -326,3 +391,29 @@ def check(self, instance):
             if m in self.RATES:
                 m = self.normalize(m.lower(), 'mongodb') + "ps"
                 self.rate(m, value, tags=tags)
+
+        stat_metrics = filter(lambda x: x.startswith('stats.'), self.METRICS)
+        for s in dbstats:
+            for m in stat_metrics:
+                value = dbstats[s]
+                dbtag = s  # key is db name, so we use that as the tag
+                try:
+                    for c in m.split('.'):
+                        value = value[c]
```

Comment:
I cleaned it up a bit following your advice, but I need the try...except: because apparently not all metrics are always reported by mongo. At least nsSizeMB isn't :(, I'll keep the try...except block in for now, if that's OK.